resources/metabase-plugin.yaml

```diff
@@ -17,9 +17,19 @@ driver:
           placeholder: Database
     - user
     - password
-    - merge:
-        - additional-options
-        - placeholder: "engine=engine_name"
+    - name: account
+      display-name: Account name
+      required: false
+      placeholder: account name
+    - name: engine_name
+      display-name: Engine name
+      required: false
+      placeholder: engine name
+      helper-text: If not specified, the default engine will be picked.
```

Comment:
Do we want to add engine_url support as well?

Reply:
I don't think so (it is getting deprecated)